### DISCUSSION TITLE
Multiple inline lightboxes with "finite: true" cause automatic scrolling to the last lightbox on page loaded

### DISCUSSION THREAD

**Discussion by laukaichung** [igordanchenko/yet-another-react-lightbox#144] [Q&A] [answered]
I have multiple inline lightboxes on the same page. When the page is initially loaded, the scrolling would automatically go to the last of the lightboxes. I believe setting `finite` to true is the culprit. Setting it to false, the problem is gone. Is it a bug?

`finite:true` example:

https://codesandbox.io/p/sandbox/yet-another-react-lightbox-136-forked-xp9j8k?file=%2Fsrc%2FApp.tsx%3A20%2C47

```jsx
<Lightbox
  carousel={{
    finite: true,
    spacing: 0,
    padding: 0,
    preload: 3,
  }}
  index={index}
  slides={slides}
  plugins={[Inline]}
  inline={{
    style: {
      width: "100%",
      maxWidth: "900px",
      aspectRatio: "3 / 2",
      margin: "0 auto",
    },
  }}
  on={{
    view: ({ index }) => setIndex(index),
  }}
/>
```

**Comment by igordanchenko** [comment, marked as answer]
This is definitely a bug. Thank you for the detailed bug report! Please follow #145 for updates.
